src/genesis/observers/accounts.py

```diff
@@ -10,9 +10,15 @@
 from src.genesis.db import DBTypes, TableManager
 from src.genesis.state import Balance
 from .chain_id import ChainIdObserver
+from psycopg.errors import UniqueViolation
+
+from tests.helpers.field_enums import Accounts
```

Comment:
@MissingNO57 let's promote this to library code (as opposed to test code) if we're going to be using it in library code. I believe this was originally your suggestion. I think it makes sense to do straight away. 

I suppose this is a matter of moving the field_enums file and updating all the references. If you're up for it, I think that would be best as a separate PR. I'm happy to help with any related git gymnastics if you like.

Reply:
Sure, I will do it.